### PR TITLE
Add Software Development Lifecycle soapbox

### PR DESCRIPTION
**Pull request by rbriski**
At Raybeam, we have a variety of ways that we run a project but we try to follow a template.  This is that template.  We pick what works for us and leave what doesn't but it's nice to have a place to start from.

### CODE REVIEW

**Comment by theothermattm** [comment]
Thanks for contributing and congrats on being the first pull request :-D

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 - [Soapboxes](#soapboxes)
   - [Angular](#angular)
   - [General Software Development Best Practices](#general-software-development-best-practices)
+  - [Software Development Lifecycle Template](soapbox/sdlc/how_we_work.md)
```
Comment (theothermattm) [inline README.md:17]: I think let's put this under the "General Software Development Best Practices" section under a "How Raybeam's Software Development Process Works" section or something like that.  Open to ideas but we want the naming to show that this is one person/group's opinion.

**Comment by rbriski** [comment]
I've updated the location.

**Comment by theothermattm** [comment]
Thanks @rbriski !

**Pull request merged**